dev/tasks/python-wheels/travis.osx.yml

```diff
@@ -38,17 +38,24 @@ before_install:
   - git -C arrow fetch -t {{ arrow.remote }} {{ arrow.branch }}
   - git -C arrow checkout {{ arrow.head }}
 
+  # ARROW-3976 Old versions of git can cause failures when Homebrew prints a
+  # donation solicitation. Attempt to update git
+  - git --version
+  - brew upgrade git
+
   # Also remove artifacts that depend on Boost
   - brew uninstall boost cgal postgis sfcgal
   - brew update
   - brew upgrade cmake
-  - brew install bison flex llvm@7
+  - brew install bison flex grpc openssl llvm@7
```

Comment:
I'm curious: why don't we use `cpp/Brewfile`?

Reply:
One reason is because the Brewfile includes Boost and this builds a privately namespaced Boost to avoid conflicts